lib/ansible/modules/cloud/amazon/_ec2_ami_search.py

```diff
@@ -95,7 +95,7 @@
 
 import csv
 import json
-import urlparse
+import ansible.module_utils.six.moves.urllib_parse as urlparse
```

Comment:
This one it looks like urlparse isn't used in the module so we can simply get rid of it.